query/influxql/transpiler.go

```diff
@@ -735,7 +734,7 @@ func (t *transpilerState) from(m *influxql.Measurement) (ast.Expression, error)
 								Name: "bucketID",
 							},
 							Value: &ast.StringLiteral{
-								Value: mapping.BucketID.String(),
+								Value: mappings[0].BucketID.String(),
```

Comment:
I'm not familar with the dbrp mapping concept. Is it safe to assume in this context that the first object is the desired one?

Reply:
I believe the original implementation `DBRPMappingService.Find` did the same thing. It just returned the first item it found for the search filter:
https://github.com/influxdata/influxdb/blob/b3dd6fead48904bd9329860720e382caf9d5690d/inmem/dbrp_mapping_service.go#L94

The new V2 versions now just supports doing a find many and picking _an answer_. I believe because of the composite key of DB and RP.

Reply:
Ah, okay. I think I get that.